tests/unit/semantics/types/test_pure_types.py

```diff
@@ -76,35 +76,33 @@ def test_valid_literals(build_node, type_):
     sources = VALID_LITERALS[type_]
     for source in sources:
         node = build_node(source)
-
         do_validate_node(type_, node)
 
 
 @pytest.mark.parametrize("type_", TYPES.keys())
 @pytest.mark.parametrize("source", INVALID_LITERALS)
 def test_invalid_literals(build_node, type_, source):
-    node = build_node(source)
     with pytest.raises((InvalidLiteral, OverflowException, UnexpectedNodeType)):
+        node = build_node(source)
         do_validate_node(type_, node)
```

Comment:
```suggestion
        build_node(source)
```
I think `build_node` already throws.

Reply:
nice catch, thanks